bpo-44682: Handle invalid arg to pdb's "commands" directive (#27252)
python/cpython: Misc/NEWS.d/next/Library/2021-07-20-00-11-47.bpo-44682.3m2qVV.rst

> In :mod:`pdb`, *commands* directive was fixed to disallow setting commands

Reviewer: ```suggestion
Change the :mod:`pdb` *commands* directive to disallow setting commands
```

> for an invalid breakpoint and to display appropriate error.

Reviewer: ```suggestion
for an invalid breakpoint and to display an appropriate error.
```

> for a non-existing breakpoint and to display "out of range" error.

Reviewer: It's not always "out of range" - if you look at Bdb.get_bpbynumber there are a number of other possible error messages.
